bcipy/parameters/parameters.json

```diff
@@ -863,7 +863,7 @@
   },
   "backspace_always_shown": {
     "value": "false",
-    "section": "bci_config",
+    "section": "aaaa_booleans",
```

Comment:
Let's name this something else... like, `top_level_config`? something a little more descriptive.